### PR TITLE
Feat/graphql docs

### PR DESCRIPTION
**Pull request by Kanakala**

### CODE REVIEW

**Comment by Kanakala** [comment]
I used this docusarous [plugin](https://www.npmjs.com/package/@edno/docusaurus2-graphql-doc-generator) to generate graphql docs. 

Screenshot 2022-03-15 at 5 54 22 PM

File: package.json [hunk 1]
```diff
@@ -4,23 +4,27 @@
...
-    "write-translations": "docusaurus write-translations",
-    "write-heading-ids": "docusaurus write-heading-ids",
+    "start": "npm run graphql:generate && docusaurus start",
+    "build": "npm run graphql:generate && docusaurus build",
+    "swizzle": "npm run graphql:generate && docusaurus swizzle",
+    "deploy": "npm run graphql:generate && docusaurus deploy",
+    "clear": "npm run graphql:generate && docusaurus clear",
+    "serve": "npm run graphql:generate && docusaurus serve",
+    "write-translations": "npm run graphql:generate && docusaurus write-translations",
+    "write-heading-ids": "npm run graphql:generate && docusaurus write-heading-ids",
```
Comment (anttiviljami) [inline package.json:14]: do we really have to generate graphql on _every_ command? 😅

File: docusaurus.config.js [hunk 2]
```diff
@@ -2,224 +2,249 @@
...
+          ],
+          copyright: `&copy; epilot.cloud ${new Date().getFullYear()}`,
+        },
+        prism: {
+          theme: lightCodeTheme,
+          darkTheme: darkCodeTheme,
+        },
+      }),
+  }
+);
```
Comment (anttiviljami) [inline docusaurus.config.js:250]: Is there any chance you could remove the formatting changes so we can see the actual changes? 🙏

File: package.json (hunk 1 above)
Comment (Kanakala) [reply package.json:14]: Yes, I don't think so. I just wanted to hear from you 😅 What about having it on below?

- npm start
- npm build
- npm deploy

File: docusaurus.config.js (hunk 2 above)
Comment (Kanakala) [reply docusaurus.config.js]: @anttiviljami Just disabled the prettier on the config file for now. Will format the file properly in a separate PR.

File: scripts/generate-graphql.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+rm -rf graphql
```
Comment (anttiviljami) [inline scripts/generate-graphql.sh:3]: This is a bit dangerous ;) 

Maybe start with 

```sh
BASEDIR=$(dirname "${0}/..")
rm -rf $BASEDIR/graphql
```

File: static/graphql/customer-portal.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+slug: /customer-portal
+title: Customer Portal
+sidebar_position: 1
+hide_table_of_contents: true
+pagination_next: null
+pagination_prev: null
+sidebar_class_name: navbar__toggle
+---
+
+This documentation has been automatically generated using [Customer Portal Graphql](https://customer-portal-api.dev.sls.epilot.io/v1/graphql) endpoint.
```
Comment (anttiviljami) [inline static/graphql/customer-portal.md:12]:
```suggestion
This documentation has been automatically generated using [Customer Portal Graphql](https://customer-portal-api.sls.epilot.io/v1/graphql) endpoint.
```

Only production APIs in docs please

**Review by anttiviljami — APPROVED** [review]
LGTM ✅

**Pull request merged**